launch/zed_test.launch

```diff
@@ -21,14 +28,13 @@
           args="manager" output="screen"/>
 
   <include file="$(find mrover)/launch/zed.launch">
-    <arg name="use_rtabmap_stereo_odom" value="$(arg use_rtabmap_stereo_odom)"/>
     <arg name="use_builtin_visual_odom" value="$(arg use_builtin_visual_odom)"/>
   </include>
 
-    <node if="$(arg run_object_detector)"
+    <!-- <node if="$(arg run_object_detector)"
```

Comment:
Instead of commenting this out just add `run_object_detector:=false` to the end of your `roslaunch` command

Reply:
👍